rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-neutron-venv.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: verify it's OK to check for errors, and that it won't cause
+# this playbook to fail in the case of failures happening under normal circumstances
+
+- name: Find running neutron services not in venv
+  shell: |
+    pgrep -a "neutron" | awk '{print $2,$3}' | grep -vP "{{ neutron_venv_bin }}/\w+[\d\.\d]?\s{1}{{ neutron_venv_bin }}/\w+"
+  register: neutron_output
+  failed_when: "neutron_output.stdout_lines|length != 0"
+
+- name: Display output of neutron_output
+  debug: var=neutron_output
```

Comment:
@mattt416 , The debug statement wouldn't print without ansible running in debug, but the point would be to get the full verbose output of it when running in debug. I threw it in there for a similar flow to the other tasks, such as [as the neutron agent check](https://github.com/rcbops/rpc-openstack/blob/fc40fa13146d4e685e97733bcd495dd38a61eac0/rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-utility.yml#L43-L45)